test/Bugs/randombug.baseline

```diff
@@ -63,10 +63,10 @@ Test case 30
 Passed
 Test case 31
 Error
-	at func2 (randombug.js:287:9)

```

Comment:
I'll mark this as Won't Fix and follow up on this in the future. As I work on Intl these tests will likely evolve and I'll probably revisit some issues like this.

---
In reply to: [107526651](https://github.com/Microsoft/ChakraCore/pull/2726#discussion_r107526651) [](ancestors = 107526651)